app/controllers/images_controller.rb

```diff
@@ -1,12 +1,16 @@
 class ImagesController < ApplicationController
   before_action :require_user
   include Glitchable
+  include S3able
+
   def create
-    image = current_user.api_images.create!(image_params)
-    download = Down.download(image.url)
-    image.original.attach(io: download, filename: "#{image.url}.jpg")
-    basic_glitch(image)
-    redirect_to search_show_path(image)
+    api_image = current_user.api_images.create!(image_params)
```

Comment:
This controller action is still quite fat, may need some refactoring down the line.